### PR TITLE
feat: Normalise paths

### LINKED ISSUE #9: URLs should be normalised to remove full path
Like [previously](https://github.com/getsentry/sentry-electron/blob/31312ce0ef91aaed2c8e03e154e9aee8bbdd1c05/src/electron-sentry.ts#L276), URLs need normalising to the app base so Sentry can group them correctly.

The previous regex only catered for apps packaged in asar.

### PR DESCRIPTION
**Pull request by timfish**
Closes #9

### CODE REVIEW

File: src/lib/electron.ts [hunk 1]
```diff
@@ -498,4 +500,44 @@ export class SentryElectron implements Adapter {
...
+      event.culprit = this.normalizeUrl(event.culprit, appPath);
+    }
+
+    if (event.request && event.request.url) {
+      event.request.url =
+        this.normalizeUrl(event.request.url.replace('file:///', ''), appPath.replace(/\\/g, '/'));
+    }
+
+    const stacktrace =
+      event.stacktrace
```
Comment (timfish) [inline src/lib/electron.ts]: Is `event.stacktrace` to find the stacktrace in older browsers? Is this even required in Electron?

File: src/lib/electron.ts [hunk 2]
```diff
@@ -498,4 +500,44 @@ export class SentryElectron implements Adapter {
...
   }
+
+  private normalizeEvent(event: any) {
+    const appPath = app.getAppPath();
+
+    if (event.culprit) {
+      event.culprit = this.normalizeUrl(event.culprit, appPath);
+    }
+
+    if (event.request && event.request.url) {
```
Comment (timfish) [inline src/lib/electron.ts]: Is `event.request.url` normalisation required to properly fingerprint in Sentry?

Comment (HazAT) [reply src/lib/electron.ts]: If there is a stacktrace, the the frames of it will be used for the fingerprint, so I would keep it like this.

**Comment by HazAT** [comment]
BTW
Feel free to clone this repo and create PRs directly here. In case we want to add commits to your branch.

**Comment by HazAT** [comment]
https://forum.sentry.io/t/replacing-culprits-with-transaction-names/105
`culprit` is called transaction now

**Comment by timfish** [comment]
@HazAT This option is ticked by default so you should still be able to add commits to my branch: 

image

**Comment by timfish** [comment]
It looks like both `@sentry/node` and `@sentry/browser` are calling `send()` with events containing `culprit` so it needs fixing in both of those and then here?

**Comment by timfish** [comment]
This is going to require more work. This current normalising works on the example app but fails on a couple of production apps I've tested. 

I think this difference is loading JavaScript source through `require` (ie. nodejs) and through HTTP requests. 

When I run the example app and trigger a JavaScript exception in the renderer process I get:
```
culprit: 'C:\\Users\\tim\\Documents\\Repos\\sentry-electron\\example\\renderer.js'
```
When I do the same in a customers app in dev mode I get:
```
culprit: 'file:///C:/Users/tim/Documents/Repos/PicoLog6/dist/layout/dev-menu.js',
```

**Comment by timfish** [comment]
@HazAT are you happy with this as it is for now?

We just need to remember to include tests that throw exceptions in resources loaded via both `require` and `file://` as the paths vary.

**Pull request merged**